compatibility/bazel_tools/daml_script/daml_script.bzl

```diff
@@ -57,6 +57,20 @@ def daml_script_test(compiler_version, runner_version):
     daml_runner = "@daml-sdk-{version}//:daml".format(
         version = runner_version,
     )
+    use_canton = versions.is_at_least("2.0.0", runner_version) and versions.is_at_least("1.16.0", compiler_version)
```

Comment:
maybe worth adding a comment that 1.16 is the one that defaulted to LF 1.14?

Reply:
Added!